packages/loot-core/src/server/accounts/rules.ts

```diff
@@ -200,10 +200,11 @@ const CONDITION_TYPES = {
       'isNot',
       'doesNotContain',
       'notOneOf',
+      'subExpression',
```

Comment:
This works! I think this `op` can best be called `and` to be most clear

```suggestion
      'and',
```

Reply:
Yeah, that's a good point. I could see it being generalized if necessary, but it definitely is not right now, since it only uses and.